README.md

```diff
@@ -1,9 +1,19 @@
 # Lawnicons
 
-[![Build debug APK](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_debug_apk.yml/badge.svg)](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_debug_apk.yml)
-[![Build release APK](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_release_apk.yml/badge.svg)](https://github.com/LawnchairLauncher/lawnicons/actions/workflows/build_release_apk.yml)
```

Comment:
Is there any reason to delete it?

I can imagine that developers use this badge to get to the "Build release APK" section faster.

Reply:
> Is there any reason to delete it?

No, just forgot to add it back in 😅